vmdb/Gemfile

```diff
@@ -11,9 +11,13 @@ gem "jquery-rjs", "=0.1.1", :git => 'https://github.com/amatsuda/jquery-rjs.git'
 gem 'angularjs-rails', '=1.2.4'
 gem 'jquery-rails', "=2.1.4"
 
+gem "win32-process", "~>0.8.0", :require => false, :platforms => [:mswin, :mingw]
```

Comment:
@djberg96 does the :platform key cause the win32-process gem to land in the Gemfile.lock for non-windows platforms?  We don't want to go to rubygems.org/yum just for a gem/rpm's specification of a gem we don't need on linux/mac.  This is why we use the `ENV['APPLIANCE']` thing like [here](https://github.com/ManageIQ/manageiq/blob/b6ed28ab035ff0e46d6b5c205e9a33f994498b58/vmdb/Gemfile#L77-L81)